src/databricks/labs/ucx/install.py

```diff
@@ -482,6 +482,13 @@ def warehouse_type(_):
                     )
                     instance_profile, spark_conf_dict = self._get_ext_hms_conf_from_policy(cluster_policy)
 
+        use_policy = self._question("Do you want to follow a policy to create clusters?", default="no")
```

Comment:
Too much code in this method, and module. Please extend with a component class. (see `configure.py`)